rtcpeerconnection.js

```diff
@@ -241,7 +241,14 @@ PeerConnection.prototype._role = function () {
 // Add a stream to the peer connection object
 PeerConnection.prototype.addStream = function (stream) {
     this.localStream = stream;
-    this.pc.addStream(stream);
+    stream.getTracks().forEach(
+        function(track) {
+            this.pc.addTrack(
```

Comment:
IIRC this was problematic in chrome. It only allows calling addTrack once? It's been a while since I used this. @jensengar do you remember?

Reply:
i've been using this in production for over a year